src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java

```diff
@@ -683,21 +718,27 @@ private static void checkRegulatingTerminal(Network network, String equipmentId)
                 .collect(Collectors.toList());
         }
 
+        public List<LfSensitivityFactor> getFactorsForContingencies(List<String> contingenciesIds) {
```

Comment:
What this method is doing? The name is not clear enough for me: too many `s` and this `for` in the middle.